lib/netsuite/actions/get_all.rb

```diff
@@ -57,7 +61,7 @@ def get_all(credentials = {})
             if response.success?
               response.body.map { |attr| new(attr) }
             else
-              raise RecordNotFound, "#{self} with OPTIONS=#{options.inspect} could not be found"
+              raise RecordNotFound, response.body.message
```

Comment:
This approach is different from the class-level methods like `search` and `get_list`. Those methods return false if the operation is not successful instead of raising an exception. Also, `RecordNotFound` isn't the correct error here: users may be leveraging that error class for situations where the record doesn't actually exist in the NS instance.

The issue with that approach is the errors are not exposed to the user for class-level operations.

For now, let's keep with the current convention. Later on we can rework how class-level operations work and expose the errors to the user in a useful wayl.